src/main/java/com/oracle/nosql/spring/data/core/mapping/NosqlTable.java

```diff
@@ -128,4 +128,35 @@
      * the set here.
      */
     int timeout() default Constants.NOTSET_TABLE_TIMEOUT_MS;
+
+    /**
+     * Sets the default table level Time to Live (TTL) when table is created.
+     * TTL allows automatic expiration of table rows when specified duration of
+     * time is elapsed. If not set the value
+     * {@link com.oracle.nosql.spring.data.Constants#NOTSET_TABLE_TTL} is used,
+     * which means no table level TTL. This is applicable only when
+     * {@link #autoCreateTable} is set to true.
+     * <p>
+     *
+     * @since TODO
```

Comment:
@cezarfx Which version to use for @since

Reply:
Let's do 1.5.0